slsReceiverSoftware/src/Listener.cpp

```diff
@@ -184,6 +184,14 @@ void Listener::ShutDownUDPSocket() {
     }
 }
 
+void Listener::DeleteUDPSocket() {
+    if (udpSocket) {
+        udpSocket = nullptr;
```

Comment:
udpSocket.reset() might be more clear but would to the same thing.

Reply:
done